ocd-3018.sql

```diff
@@ -0,0 +1,131 @@
+do $$
+begin
+	declare addr_count integer;
+
+	--Is addr_id 1 still tied to multiple agencies
+	select count(*) into addr_count from (
+		select address_id from openchpl.vendor where deleted = false
+		union all 
+		select address_id from openchpl.certification_body where deleted = false
+		union all 
+		select address_id from openchpl.testing_lab where deleted = false) addr_cnt
+	where address_id = 1
+	group by address_id;
+
+	if addr_count > 1 then
+		insert into openchpl.address (street_line_1, street_line_2, city, state, zipcode, country, last_modified_user)
+		select street_line_1, street_line_2, city, state, zipcode, country, -1
+		from openchpl.address
+		where address_id = 1;
+		
+		update openchpl.testing_lab
+		set address_id = lastval(), last_modified_user = -1
+		where vendor_id = 1;
```

Comment:
Does this update statement work? Updating testing_lab where vendor_id =1 doesn't seem right.